src/main/java/com/google/opensesame/user/UserData.java

```diff
@@ -60,6 +63,9 @@ public UserData(UserEntity userEntity) throws IOException {
     image = userGitAccount.getAvatarUrl();
     location = userGitAccount.getLocation();
     name = userGitAccount.getName();
+    if (name == null) {
+      name = gitHubID;
+    }
```

Comment:
Could we instead use `name = userGitAccount.getLogin()` here? We know that a user will have a GitHub username, and that seems like a much more human-readable backup than using the numerical ID.

[Here](https://github-api.kohsuke.org/apidocs/org/kohsuke/github/GHPerson.html#getLogin()) is the Java GitHub API reference.

Reply:
Okay just realized that we are storing a user's login as their `gitHubId`. Could we document this or change the naming to be more consistent with GitHub's nomenclature?

Reply:
Yeah, the Login is the only identifier we get from the API. Sadly, the nomenclature has been widely adopted across our website. To improve consistency with the GitHub API, I'll try to change all the names. This might have adverse effects on the already rolled out website though. We might have to clear Datastore.